### PR TITLE
Updated README and Asset catalog housekeeping

### PR DESCRIPTION
**Pull request by jamie-brannan**
The following is a response to issues #27 and a request to add assets from #23 during Hacktoberfest 2020 🎃 

As a maintainer, have this PR count in accordance with the [latest update](https://hacktoberfest.digitalocean.com/hacktoberfest-update), please opt-in by...

- [x] assigning `@jamie-brannan` to PR

- [ ] Add the label `hacktoberfest-accepted` to the PR

Thank you for the opportunity to participate and build things together 🎉

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,21 +14,26 @@ The app is currently in the second stage of development and several new features
 
 The project uses test-drive agile development processes, where requirements are turned into test cases, and code is improved upon. 
 
+#### Table of Contents
```
Comment (pushpinderpalsingh) [inline README.md:17]: Remove this

File: README.md [hunk 2]
```diff
@@ -14,21 +14,26 @@ The app is currently in the second stage of development and several new features
...
+## 👾 Technologies
+> To help scale development, this app is built using the MVC design pattern. subsequent changes made by contributors do not affect the central architecture.
 * Fetches and renders JSON from the SpaceX API
 * Intuitive error handling
 * Animations via the Lottie framework
 * Unit testing using XCTests framework
 
 
-## [](https://github.com/pushpinderpalsingh/SpaceDash#screenshots)Screenshots 📸 
+## 📸 Screenshots
```
Comment (pushpinderpalsingh) [inline README.md:32]: Needs to be updated

File: README.md [hunk 3]
```diff
@@ -43,27 +48,36 @@ The project uses test-drive agile development processes, where requirements are
...
+
+Before you start developing please read through [Contributing.md](https://github.com/pushpinderpalsingh/SpaceDash/blob/develop/Contributing.md) 
+
+#### Set up contributing environment
+
+Before you assist with the development, please set up the project on your local machine, run it, and go through the complete application 
+
+Click through buttons you can find and see where they take you. Explore the app in detail so that you are aware of core features.
 
+#### Issues
```
Comment (pushpinderpalsingh) [inline README.md:67]: I think contributing.md would be a better place for this

File: README.md [hunk 4]
```diff
@@ -14,21 +14,26 @@ The app is currently in the second stage of development and several new features
...
 * Unit testing using XCTests framework
 
 
-## [](https://github.com/pushpinderpalsingh/SpaceDash#screenshots)Screenshots 📸 
+## 📸 Screenshots
 
 [![alt text](https://github.com/pushpinderpalsingh/SpaceX/raw/develop/Screenshots/collage.png "Screen Shots")](https://github.com/pushpinderpalsingh/SpaceX/blob/develop/Screenshots/collage.png)
 
-## [](https://github.com/pushpinderpalsingh/SpaceDash#getting-started-for-development)Getting Started With Development ⚙️
+## ⚙️ [](https://github.com/pushpinderpalsingh/SpaceDash#getting-started-for-development)Quick Start
```
Comment (pushpinderpalsingh) [inline README.md:36]: I believe the previous title was perfect.

**Review by pushpinderpalsingh — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (jamie-brannan) [reply README.md:36]: I'm not convinced. "Getting started with Development" sounds like an introduction to the career of being a developer. It doesn't sound like native English in this context. This is getting started using our project Space Dash as a developer, therefore we could be more specific.

I see "Quick Start", often used as a reference to the expression "quickly start your development environment". The information is all about getting the project up and running.

File: README.md (hunk 3 above)
Comment (jamie-brannan) [reply README.md:67]: Meaning I should move this entire section to contributing.md?

Comment (jamie-brannan) [reply README.md:67]: Actually I just saw that there's an issues section that already exists! I'll combine them 🚀

File: README.md (hunk 4 above)
Comment (pushpinderpalsingh) [reply README.md:40]: Then rename to just Getting Started.

**Review by pushpinderpalsingh — CHANGES_REQUESTED** [review]
@jamie-brannan there is still a Screenshots folder in your branch. Maybe move those and update the link in Architecture.md? 

I believe the QR code is way too big for the readme. Can you reduce that?

Under screenshots there are no screenshots appearing on readme!

**Comment by jamie-brannan** [comment]
Hi, yes i should be able to reduce and adjust that. Could you please send me some screen shots of what view you're looking at however so I may have a point of reference.

Clearly there are no screenshots placed – this is because I believed Screenshots were being selected still and placed in the related issue from which I pulled screenshots. 

I'll resize and add the related elements but it would help to have a reference point for the display size you're looking for and insure we're looking at markdown that's been interpreted the same way.

**Comment by pushpinderpalsingh** [comment]
I believe the display size and overall size of screenshots was perfect with previous screenshots. You can refer to my fork for reference.

**Comment by jamie-brannan** [comment]
Hello @pushpinderpalsingh 👋  I'm just now pushing through several commits that should address the remaining issues (section title, Screenshots, and image asset dynamic display width) with the `README.md`. I'll rebase my branch on the current develop and it should be good to go.

I'll check in again for any additional remarks, but I believe this PR is ready to go! 🚀

**Comment by pushpinderpalsingh** [comment]
Thanks for your contribution!

**Comment by jamie-brannan** [comment]
You're welcome @pushpinderpalsingh ! 👉  Could you please add the label `hacktoberfest-accepted` to the PR however so it counts towards the event? 🙏  🙏

**Comment by pushpinderpalsingh** [comment]
Sure thing

**Pull request merged**